docs/adr/adr-030-stdin-sync.md

```
@@ -0,0 +1,139 @@
+                         std::istreambuf_iterator<char>());
+}
+
+if (!context.empty()) {
+  cfg.prompt = cfg.prompt.empty() ? context : cfg.prompt + "\n\n" + context;
+  cfg.mode = Mode::Sync;
+}
+```
+
+### CLI help update
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)